shakedown/dcos/command.py

```diff
@@ -52,7 +52,7 @@ def run_command(
         try_close(channel)
         try_close(transport)
 
-        return True
+        return exit_code == 0
```

Comment:
Now that we have an exit code, should we just pass it back to the caller?

/cc @sschneid